jmeter/src/jmeter/load-test-http-router-docker.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
```

Comment:
this should be just 2019 as you have newly added the file, right?

Reply:
Changed.